pro/src/screens/OfferEducational/OfferEducationalForm/OfferEducationalForm.tsx

```diff
@@ -97,6 +119,16 @@ const OfferEducationalForm = ({
     getIsOffererEligibleToEducationalOfferAdapter,
   ])
 
+  useEffect(() => {
+    if (submitCount > previousSubmitCount && !isValid) {
```

Comment:
Pourquoi avoir besoin du `submitCount`?
Là comme ça j'aurais tendance à dire que si le form n'est pas valid, alors on `scrollToFirstError`.
Pourquoi ce check sur `submitCount`?

Reply:
`isValid` est false dès le début puisque tous les champs ne sont pas remplis, du coup, si je ne check pas qu'on a soumis le formulaire en + de l'état `isValid` alors on n'a pas le scroll to first error qui s'exécute

Ensuite, je check qu'on a soumis le formulaire une nouvelle fois pour éviter de re-scroller à la prochaine erreur dès que tu as rempli un champs :)

Reply:
tu as pas besoin de tout ça je pense... 

en checkant si `isSubmitting && !isValid`, du coup, plus besoin de faire un `setPreviousSubmitCount(submitCount)` et de hack eslint... à tester